Philips.CodeAnalysis.Test/Verifiers/DiagnosticVerifier.cs

```diff
@@ -19,6 +22,9 @@ namespace Philips.CodeAnalysis.Test.Verifiers
 	public abstract partial class DiagnosticVerifier
 	{
 		#region To be implemented by Test classes
+
+		protected virtual DiagnosticId DiagnosticId { get; } = DiagnosticId.None;
```

Comment:
Or shall the initial value be: GetDiagnosticsAnalyzer().SupportedDiagnostics.First().Id ?

Reply:
Yes, good idea. A couple hoops because this is a string rather than a DiagnosticId, but it works well. Done.